test/uk/gov/hmrc/apigatekeeperxmlservicesfrontend/views/organisation/OrganisationDetailsViewSpec.scala

```diff
@@ -104,13 +142,13 @@ class OrganisationDetailsViewSpec extends CommonViewSpec {
       document.getElementById("user-email-0").text() shouldBe "a@b.com"
       document.getElementById("user-link-0").attr(
         "href"
-      ) shouldBe s"https://admin.qa.tax.service.gov.uk/api-gatekeeper/developer?developerId=${organisationUsers.head.userId.value}"
+      ) shouldBe s"https://admin.qa.tax.service.gov.uk/api-gatekeeper/developer?developerId=${organisationUsers.head.userId.get.value}"
```

Comment:
I wonder if using ScalaTests option values trait can make this code not need the .get everywhere?
https://www.scalatest.org/user_guide/using_OptionValues

Reply:
Like this?